diff --git a/node_modules/gologin/src/gologin.js b/node_modules/gologin/src/gologin.js
index e9cea07..5bb3a90 100644
--- a/node_modules/gologin/src/gologin.js
+++ b/node_modules/gologin/src/gologin.js
@@ -77,7 +77,7 @@ export class GoLogin {
     this.timezone = options.timezone;
     this.extensionPathsToInstall = [];
     this.customArgs = options.args || [];
-    this.restoreLastSession = options.restoreLastSession || true;
+    this.restoreLastSession = options.restoreLastSession ?? true;
     this.processSpawned = null;
     this.processKillTimeout = 1 * 1000;
     this.browserMajorVersion = 0;
